src/main/resources/wa-task-configuration-employment-et_englandwales.dmn

```diff
@@ -311,7 +311,7 @@ else "1234"</text>
           <text>"description"</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_02zdiu5">
-          <text>"[Review the Referral](/cases/case-details/${[CASE_REFERENCE]}/createReferral1)"</text>
+          <text>"[Review the Referral](/cases/case-details/${[CASE_REFERENCE]}/trigger/createReferral/createReferral1)"</text>
```

Comment:
Hi Kala,
Are we sure that this is the correct response? If someone receives a Referral, they may not be looking to create a Referral immediately, they may want to be brought to the Referral Tab to inspect the Referral that generated the task.